src/routes/Evodex/BackLayer/Exchange/ExchangeBackLayer.js

```diff
@@ -513,6 +523,8 @@ const ExchangeBackLayer = ({
             </Typography>
           }
           useHelperTextAsNode
+          decimalScale={4}
```

Comment:
I think we may need more than 4 decimal places

@sergioyuhjtman  what is the maximum number of decimals we should set?

![image](https://user-images.githubusercontent.com/5632966/95001681-71a18000-0589-11eb-92ce-0c0d428555e1.png)